---
layout: page
lang: pl
title: Badania i publikacje
order: 300
---

Cheynna Crowley, Yuchen Yang, Yunjiang Qiu, Benxia Hu, Armen Abnousi, __Jakub Lipiński__, Dariusz Plewczyński, Di Wu, Hyejung Won, Bing Ren, Ming Hu, Yun Li,

Computational and Structural Biotechnology Journal 2020, ISSN 2001-0370

[FIREcaller: Detecting Frequently Interacting Regions from Hi-C Data](http://www.sciencedirect.com/science/article/pii/S2001037020305511)
[![Archived PDF](/assets/images/pdf.svg){: style="width:1em; display: inline; margin: 0 0 0rem;" }](/assets/pdf/1-s2.0-S2001037020305511-main.pdf)

<small>Abstract: Hi-C experiments have been widely adopted to study chromatin spatial organization, which plays an essential role in genome function. We have recently identified frequently interacting regions (FIREs) and found that they are closely associated with cell-type-specific gene regulation. However, computational tools for detecting FIREs from Hi-C data are still lacking. In this work, we present FIREcaller, a stand-alone, user-friendly R package for detecting FIREs from Hi-C data. FIREcaller takes raw Hi-C contact matrices as input, performs within-sample and cross-sample normalization, and outputs continuous FIRE scores, dichotomous FIREs, and super-FIREs. Applying FIREcaller to Hi-C data from various human tissues, we demonstrate that FIREs and super-FIREs identified, in a tissue-specific manner, are closely related to gene regulation, are enriched for enhancer-promoter (E-P) interactions, tend to overlap with regions exhibiting epigenomic signatures of cis-regulatory roles, and aid the interpretation or GWAS variants.</small>

[https://doi.org/10.1016/j.csbj.2020.12.026](https://doi.org/10.1016/j.csbj.2020.12.026)